src/js/shade.js

```diff
@@ -24,7 +24,7 @@
         Shade.__super__.call(this, element, options, Shade.DEFAULTS);
     }
 
-    Shade.VERSION = '0';
+    Shade.VERSION = '1.1.1';
```

Comment:
Pretty sure this line shouldn't be there in the first place, it should be added into our dist file.